azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -33,9 +33,8 @@
   DISABLED(100),
   QUEUED(110),
   FAILED_SUCCEEDED(120),
-  CANCELLED(130);
-  // status is TINYINT and in H2 DB the possible values are: -128 to 127
-  // so trying to store CANCELLED in H2 fails at the moment
```

Comment:
Oh, sorry. Probably I got confused with the max value on unsigned vs. signed tinyint.